packages/builder/src/steps/clone.ts

```diff
@@ -253,7 +253,7 @@ const cloneSpec = {
       state: builtState,
       meta: deployInfo.meta,
       status: partialDeploy ? 'partial' : 'complete',
-      chainId,
+      chainId: runtime.chainId,
```

Comment:
this is wrong. `clone` steps should be pulling from chain ID 13370 by default. This is as intended. Please undo.